api/src/test/java/org/pmiops/workbench/CallsRealMethodsWithDelay.java

```diff
@@ -1,18 +0,0 @@
-package org.pmiops.workbench;
```

Comment:
These 2 removals of unused test classes are unrelated - I just happened to notice them